### DISCUSSION TITLE
Hiding widget of individual additionalproperties instance using JS API

### DISCUSSION THREAD

**Discussion by ErikKoinberg** [bhch/django-jsonform#155] [Q&A] [answered]
I have an additionalproperties that are objects. Depending on what the user inputs into the first field of the additional property object, I want to hide the widget of that specific additional property object. How can this be done? Changing the schema of the additional properties objects hides or shows the widget for all such objects, but I need only to hide specific ones.

**Comment by bhch** [comment]
I don't think it's possible at present.

**Comment by ErikKoinberg** [comment, marked as answer]
A workaround would be to not use defs but instead create a separate dict and reference this multiple times in the main schema dict. This way, repetition can be avoided while allowing for individual tweaking on the front end.
